tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -376,6 +376,21 @@ def test_integer_conversion_fail(self, duckdb_cursor):
         double_dtype = np.dtype('object')
         assert isinstance(converted_col['0'].dtype, double_dtype.__class__) == True
 
+    # numpy.datetime64 is just a wrapper around a datetime.datetime object, because reasons..
+    def test_numpy_datetime(self):
+        numpy = pytest.importorskip("numpy")
+
+        standard_vector_size = 2048
+        data = []
+        data += [numpy.datetime64('2022-12-10T21:38:24.578696') for x in range(standard_vector_size)]
```

Comment:
```suggestion
        data += [numpy.datetime64('2022-12-10T21:38:24.578696')] * standard_vector_size
```
Small nitpick, can be applied to the other sites as well :)